test/integration/integration_test.go

```diff
@@ -38,6 +39,8 @@ const (
 	TagIncrementalBuildScripts                = "test/sti-incremental-app-scripts"
 	TagIncrementalBuildScriptsNoSaveArtifacts = "test/sti-incremental-app-scripts-no-save-artifacts"
 	TagCleanLayeredBuildNoTar                 = "test/sti-fake-no-tar"
+	TagCleanBuildOnBuild                      = "test/sti-onbuild-trigger"
+	TagOnBuildTrigger                         = "test/sti-file-check-onbuild-trigger"
```

Comment:
I can rephrase this, I used "trigger" as the Docker documentation references the ONBUILD instruction allows for any build instruction to be registered as a "trigger", so this is referring to the ONBUILD being triggered.